dist/svg.js

```diff
@@ -705,6 +705,63 @@ SVG.extend(SVG.PathArray, {
 
     return this
   }
+  // Test if the passed path array use the same commands as this path array
+, haveSameCommands: function(pathArray) {
+    var i, il, haveSameCommands
+
+    pathArray = new SVG.PathArray(pathArray)
+
+    haveSameCommands = this.value.length === pathArray.value.length
+    for(i = 0, il = this.value.length; haveSameCommands && i < il; i++) {
+      haveSameCommands = this.value[i][0] === pathArray.value[i][0]
+    }
+
+    return haveSameCommands
+  }
+  // Make path array morphable
+, morph: function(pathArray) {
+    pathArray = new SVG.PathArray(pathArray)
+
+    if(this.haveSameCommands(pathArray)) {
+      this.destination = pathArray
+    } else {
+      this.destination = undefined
```

Comment:
is this needed? I mean - its undefined anyway. And sice every dude can do something like `undefined = true`, its rather uncommon to use `undefined` in an assignment.